src/Inputs/MaskedInput.tsx

```diff
@@ -39,17 +39,14 @@ const PERCENT_FORMAT_MASK = (s: string): string => {
 const getMaskFunction_ = (
     mask: MaskedInputPreset | ((value: string) => string),
 ): ((value: string) => string) => {
-    if (mask in MaskedInputPreset) {
-        switch (mask) {
-            case MaskedInputPreset.DOLLAR:
-                return DOLLAR_FORMAT_MASK;
-            case MaskedInputPreset.PERCENTAGE:
-                return PERCENT_FORMAT_MASK;
-            default:
-                return (): string => 'Invalid mask.';
-        }
+    switch (mask) {
+        case MaskedInputPreset.DOLLAR:
+            return DOLLAR_FORMAT_MASK;
+        case MaskedInputPreset.PERCENTAGE:
+            return PERCENT_FORMAT_MASK;
+        default:
+            return mask as (value: string) => string;
```

Comment:
This is not a good practice... there should be this check which is overwritten depending on the node environment to keep developers in check without exploding in production

Reply:
Not sure what you mean here, all I did was remove the If statement

Reply:
The if statement is in place to ensure that developers using this component don't use it incorrectly and let them know that its used incorrectly. So im saying u shouldnt remove it if the code is in a dev environment, but remove it when the code is in production... which can be done using NODE_ENV

Reply:
But typescript will do the check for you so not sure of the point?